job_spec/INSAR_MULTI_BURST.yml

```diff
@@ -0,0 +1,68 @@
+INSAR_MULTI_BURST:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 2
+        example:
+          - S1_136231_IW2_20200604T022312_VV_7C85-BURST S1_136232_IW2_20200604T022315_VV_7C85-BURST
```

Comment:
Note that we're not yet sure if this will work with the current cli in hyp3-isce2's `develop` branch.